### PR TITLE
[NEW] Video recording

### LINKED ISSUE #720: [NEW] Enable recording and sending videos directly
Version: 1.4.1 (99) 

At the moment it is not possible to record and send a video directly. We have to go through the default camera app to record a video to the gallery and can then send it through the rocket.chat native app.

It would be super cool if we can record and send it directly like it works for photos.

Thanks!

### PR DESCRIPTION
**Pull request by cardoso**
@RocketChat/ios

Closes #720

### CODE REVIEW

File: Rocket.Chat/pt-BR.lproj/Localizable.strings [hunk 1]
```diff
@@ -102,6 +102,7 @@
 "chat.upload.uploading_video" = "Enviando video";
 "chat.upload.uploading_file" = "Enviando \"%@\"";
 "chat.upload.take_photo" = "Tirar uma foto";
+"chat.upload.shoot_video" = "Filmar um video";
```
Comment (rafaelks) [inline Rocket.Chat/pt-BR.lproj/Localizable.strings]: "Gravar um vídeo"?

File: Rocket.Chat/Controllers/Chat/ChatControllerUploader.swift [hunk 2]
```diff
@@ -19,6 +19,14 @@ extension ChatViewController: UIImagePickerControllerDelegate, UINavigationContr
             alert.addAction(UIAlertAction(title: localized("chat.upload.take_photo"), style: .default, handler: { (_) in
                 self.openCamera()
             }))
+
+            alert.addAction(UIAlertAction(title: localized("chat.upload.shoot_video"), style: .default, handler: { (_) in
+                AVAudioSession.sharedInstance().requestRecordPermission { granted in
+                    if granted {
+                        self.openCamera(video: true)
+                    }
```
Comment (rafaelks) [inline Rocket.Chat/Controllers/Chat/ChatControllerUploader.swift]: If not, let's show an error message?

Comment (cardoso) [reply Rocket.Chat/Controllers/Chat/ChatControllerUploader.swift]: Looks like camera handles that for us, so I'm removing this part :)

File: Rocket.Chat/pt-BR.lproj/Localizable.strings (hunk 1 above)
Comment (cardoso) [reply Rocket.Chat/pt-BR.lproj/Localizable.strings]: 👍

**Pull request merged**